packages/zosjobs/src/DownloadJobs.ts

```diff
@@ -82,10 +82,7 @@ export class DownloadJobs {
             }
             await DownloadJobs.downloadSpoolContentCommon(session, {
                 jobFile: {...file, ddname: uniqueDDName},
-                outDir: parms.outDir,
-                omitJobidDirectory: parms.omitJobidDirectory,
-                binary: parms.binary,
-                record: parms.record
+                ...parms
             });
```

Comment:
Smart way to avoid having to type new properties as they come in.

I guess I do have one small concern.
What if a new property is added to `IDownloadAllSpoolContentParms` called `jobFile`?

We could flip the order in which we configure these parameters, to make sure that we don't overwrite the current `file` and `uniqueDDName` values.
```typescript
await DownloadJobs.downloadSpoolContentCommon(session, {
    ...parms
    jobFile: {...file, ddname: uniqueDDName},
});
```

---

This is just something to keep in mind for future object construction and not something we need to be super concerned about in this particular case because if how the interfaces where combined  😋